museum/testImport

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+if [ -z "$1" ]
+then
+	echo "Usage: $0 [nuke] [load] [report] <tableList>"
```

Comment:
This usage seems inconsistent with the `getopts` style used elsewhere, is that deliberate?